File: src/mocks/handlers/module3.ts
import { http, HttpResponse } from 'msw';
import { API_BASE } from '../../api/client';
import { MOCK_LANDMARKS } from '../data/module3';

export const module3Handlers = [
  http.get(`${API_BASE}/module3/landmarks/:sessionId`, () =>
    HttpResponse.json(MOCK_LANDMARKS),
  ),

  http.get(`${API_BASE}/module3/pro/:playerId`, () =>
    HttpResponse.json(MOCK_LANDMARKS),
  ),
];

File: backend/app/api/module3.py
from fastapi import APIRouter

router = APIRouter()

@router.get('/landmarks/{session_id}')
async def landmarks(session_id: str):
    return {"session_id": session_id, "frames": []}

@router.get('/pro/{player_id}')
async def pro(player_id: str):
    return {"player_id": player_id, "frames": []}
